openr/py/openr/cli/commands/kvstore.py

```diff
@@ -218,7 +218,7 @@ def _parse_nodes(rows, value):
 
             marker = '* ' if prefix_db.thisNodeName == host_id else '> '
             row = ["{}{}".format(marker, prefix_db.thisNodeName)]
-            row.extend(prefix_strs)
+            row.extend(map(lambda l: l.split()[0], prefix_strs.splitlines()[2:]))
```

Comment:
Use newer syntax instead of lambda

```
row.extend([l.split()[0] for l in prefix_strs.splitlines()[2:]))
```

We should only print prefixes here which are of type LOOPBACK and use `utils.sprint_prefix` function to convert ip-prefix thrift object to string representation

```
// Get loopback prefixes
loopback_prefixes = [p.prefix for p in prefix_db.prefixEntries \
                     if p.prefixType = lsdb_types.PrefixType.LOOPBACK]

// add it to row
row.extend([utils.sprint_prefix(p) for p in loopback_prefixes])